tests/ui/lint_without_lint_pass.rs

```diff
@@ -1,41 +0,0 @@
-#![deny(clippy::internal)]
-#![feature(rustc_private)]
-
-#[macro_use]
-extern crate rustc;
-use rustc::lint::{LintArray, LintPass};
-
-#[macro_use]
-extern crate clippy_lints;
-
-declare_clippy_lint! {
```

Comment:
Instead of removing this test, we can change this to the `declare_lint` macro from rustc. The intend of this test would persist.